core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -226,6 +226,11 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     aclCache.toMap
   }
 
+  def close() {
+    zkUtils.zkClient.unsubscribeStateChanges(ZkStateChangeListener)
+    aclChangeListener.close()
```

Comment:
It seems that we need to close zkUtils. If we do that, there is no need to unsubscribe here and in aclChangeListener.